src/index.html

```diff
@@ -7,8 +7,29 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="0">
+    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html"><img src="images/logo.png" alt="logo"></a>
+      <nav>
+        <ul>
+          <li><a class="nav__link is-active" href="#Apple">Apple</a></li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item"><a class="nav__link is-active" href="#Apple">Apple</a></li>
```

Add classes and use them for styling

Reply:
@nazarmatsevych done!